odl/solvers/functional/functional.py

```diff
@@ -1268,6 +1333,186 @@ def __str__(self):
         return '{}.convex_conj'.format(self.convex_conj)
 
 
+class BregmanDistance(Functional):
```

Comment:
Wouldn't "`BregmanDivergence`" be a more correct term?

Reply:
I had the same thought initially, but the main reference (Burger et al (?)) calls it Bregman distance. Divergence leans more on the statistical side I assume, so IMO let's go with the terms the inverse problems and imaging people use.